src/cljs_http/core.cljs

```diff
@@ -29,15 +29,28 @@
           h-val (vals headers)]
     (.set (.-headers xhr) h-name h-val)))
 
+(defn apply-response-type!
+  "Takes an XhrIo object and sets response-type if not nil."
+  [xhr response-type]
+  (.setResponseType xhr
+   (case response-type
+     :arraybuffer XhrIo.ResponseType.ARRAY_BUFFER
+     :blob        XhrIo.ResponseType.BLOB
+     :document    XhrIo.ResponseType.DOCUMENT
+     :text        XhrIo.ResponseType.TEXT
+     :default     XhrIo.ResponseType.DEFAULT
+     nil          XhrIo.ResponseType.DEFAULT)))
+
```

Comment:
Could you change this to:
```
(case response-type
  :array-buffer XhrIo.ResponseType.ARRAY_BUFFER
  :blob XhrIo.ResponseType.BLOB
  :document XhrIo.ResponseType.DOCUMENT
  :text XhrIo.ResponseType.TEXT
  XhrIo.ResponseType.DEFAULT)
```

Use `:array-buffer` and always fall back to default?